kodilla-stream/src/main/java/com/kodilla/stream/array/ArrayOperations.java

```diff
@@ -0,0 +1,13 @@
+package com.kodilla.stream.array;
+
+import java.util.ArrayList;
+import java.util.stream.IntStream;
+
+public interface ArrayOperations {
+    static int getAverage(int[] numbers) {
+        List<int> numbers = new ArrayList<>();
+        List<int> numbersList = IntStream.range(0, ((ArrayList) numbers).getList());
```

Comment:
range - czyli zakres. Jesli jako parametr bedzie tablica 5 elementow, zakres powinien byc `range(0,5)`, czyli `range(0, numbers.length)`

Reply:
musisz w streamie printowac na konsole kolejne elementy streamu

Reply:
http://www.deadcoderising.com/2015-05-19-java-8-replace-traditional-for-loops-with-intstreams/